examples/sandbox/index.ts

```diff
@@ -175,12 +176,6 @@ async function deviceConnected(deviceId) {
     console.error("Could not initialize PortisAdapter", e);
   }
 
-  try {
```

Comment:
Leave it auto initialize now that the event has been tested

Reply:
I took this out when the initialization was throwing an error for not having a mnemonic. Since I took out the error, you're right that we can probably have it auto initialize now.

Reply:
I put it back in, but I had to remove a `wallet.initialize()` line in the "on("change")` handler for the select at the top to avoid always popping up the mnemonic required modal.